test/python/mock/test_new_qasm_backend.py

```diff
@@ -15,13 +15,13 @@
 from qiskit import QuantumCircuit, transpile
 from qiskit.test import QiskitTestCase
 from qiskit.providers.fake_provider import FakeBogota
-from qiskit.providers.fake_provider import HAS_AER
+from qiskit.utils.backend_utils import has_aer
```

Comment:
The right place to get this object from is `qiskit.utils.optionals.HAS_AER` - I didn't even realise there was old stuff kicking around in `qiskit.utils.backend_utils`, but come Terra 0.21 we'll be able to mark it as deprecated and get rid of it.